x-pack/plugins/ml/public/application/trained_models/nodes_overview/allocated_models.tsx

```diff
@@ -97,11 +107,36 @@ export const AllocatedModels: FC<AllocatedModelsProps> = ({
       'data-test-subj': 'mlAllocatedModelsTableThroughput',
     },
     {
-      name: i18n.translate(
-        'xpack.ml.trainedModels.nodesList.modelsList.modelAvgInferenceTimeHeader',
-        {
-          defaultMessage: 'Avg inference time',
-        }
+      name: (
+        <EuiToolTip
+          display={'block'}
+          title={
+            <FormattedMessage
+              id="xpack.ml.trainedModels.nodesList.modelsList.modelAvgInferenceTimeTooltipHeader"
+              defaultMessage="Average inference time"
+            />
+          }
+          content={
+            <FormattedMessage
+              id="xpack.ml.trainedModels.nodesList.modelsList.modelAvgInferenceTimeTooltipMessage"
+              defaultMessage="If caching is enabled, fast cache hits are included when calculating the average inference time"
```

Comment:
@lcawl should this end with a `.`  ? Guess it should when we are using sentence case.